### PR TITLE
Fix C1 Task Test hangups

### PR DESCRIPTION
**Pull request by ljades**

### CODE REVIEW

File: app/models/c1_task.rb [hunk 1]
```diff
@@ -3,7 +3,6 @@
```
Comment (tstrass) [inline app/models/c1_task.rb]: I think you can get rid of these `require`s now

Comment (ljades) [inline app/models/c1_task.rb]: Not sure why the comment isn't shown as outdated after I rebased, but done.

Comment (tstrass) [inline app/models/c1_task.rb]: The line I commented on didn't change

Comment (ljades) [inline app/models/c1_task.rb]: Gotcha.

**Pull request merged**